datafusion/core/src/execution/runtime_env.rs

```diff
@@ -142,8 +126,10 @@ impl Default for RuntimeEnv {
 pub struct RuntimeConfig {
```

Comment:
Not directly related to this PR, but the proliferation of `*Config` structs is a pet peeve of mine that I am working to fix under #4617 and #4349